lib/rucio/core/transfer.py

```diff
@@ -1323,12 +1323,14 @@ def __list_transfer_requests_and_source_replicas(total_workers=0, worker_number=
                                  models.Request.previous_attempt_id,
                                  models.Request.dest_rse_id,
                                  models.Request.retry_count,
-                                 models.Request.account) \
+                                 models.Request.account,
+                                 models.Request.created_at) \
```

Comment:
fyi, if this changes the returned tuple of `__list_transfer_requests_and_source_replicas` it might break the preparer https://github.com/rucio/rucio/blob/327571d841bd36d0712acf2b5ce4ae4e3ba44897/lib/rucio/core/request.py#L1530

Reply:
It doesn't change the output format. Here I change the internal subquery to propagate the field towards the main query. And the main one uses the field only for order_by